test/e2e/logs_test.go

```diff
@@ -102,7 +102,7 @@ var _ = Describe("Podman logs", func() {
 		It("tail 99 lines: "+log, func() {
 			skipIfJournaldInContainer()
 
-			logc := podmanTest.Podman([]string{"run", "--log-driver", log, "-dt", ALPINE, "sh", "-c", "echo podman; echo podman; echo podman"})
+			logc := podmanTest.Podman([]string{"run", "--log-driver", log, ALPINE, "sh", "-c", "echo podman; echo podman; echo podman"})
 			logc.WaitWithDefaultTimeout()
 			Expect(logc).To(Exit(0))
 			cid := logc.OutputToString()
```

Comment:
Whoops. Looks like you need to `run -n foo` instead and `log foo`, or find some other way to get the CID.